code/game/objects/items/weapons/implants/implanter.dm

```diff
@@ -154,66 +154,70 @@
 	update()
 
 /obj/item/implanter/ipc_tag
-	name = "IPC/Shell tag implanter"
+	name = "IPC tag implanter"
 	desc = "A special implanter used for implanting synthetics with a special tag."
-	var/obj/item/organ/internal/ipc_tag/ipc_tag = null
+	var/obj/item/organ/internal/ipc_tag/ipc_tag
 
-/obj/item/implanter/ipc_tag/New()
-	ipc_tag = new(src)
-	..()
+/obj/item/implanter/ipc_tag/Initialize()
+	. = ..()
+	ipc_tag = new /obj/item/organ/internal/ipc_tag(src)
 	update()
 
 /obj/item/implanter/ipc_tag/update()
-	if (ipc_tag)
+	if(ipc_tag)
 		icon_state = "cimplanter1"
 	else
 		icon_state = "cimplanter0"
 	return
 
-/obj/item/implanter/ipc_tag/attack(mob/M as mob, mob/user as mob)
-	if (!ishuman(M))
+/obj/item/implanter/ipc_tag/attack(mob/M, mob/user)
```

Comment:
You should probably require a aggressive grab here or them laying down, before you can tag them (ideally in combination with a do_after)

Reply:
Will implement some time in the near future, while the lore arc is done, I believe, I hope.